services/notifications.go

```diff
@@ -1526,12 +1526,26 @@ func collectAttestationAndOfflineValidatorNotifications(notificationsByUserID ma
 		}
 	}
 
-	if len(offlineValidators) > 5000 {
-		return fmt.Errorf("retrieved more than 5000 offline validators notifications: %v, exiting", len(offlineValidators))
+	var offlineValidatorsLimit int
```

Comment:
It's common to init it directly with the default.
So 
```
offlineValidatorsLimit := 5000
if ... { offlineValidatorsLimit = value }
```
is the more convenient way